community/detectors/mlflow_cve_2023_6977/src/test/java/com/google/tsunami/plugins/detectors/cves/cve20236977/Cve20236977DetectorTest.java

```diff
@@ -106,12 +106,12 @@ public void detect_whenVulnerable_returnsVulnerability() throws IOException {
                         .setMainId(
                             VulnerabilityId.newBuilder()
                                 .setPublisher("TSUNAMI_COMMUNITY")
-                                .setValue("CVE_2023_1177"))
+                                .setValue("CVE_2023_6977"))
                         .setSeverity(Severity.CRITICAL)
-                        .setTitle("CVE-2023-1177 MLflow LFI/RFI")
+                        .setTitle("CVE-2023-6977 MLflow LFI/RFI")
                         .setRecommendation(
-                            "1.Updated to version 2.2.1 or later\n2.Add authentication to MLflow "
-                                + "server\n")
+                            "1.Updated to any version above the version 2.9.2\n"
```

Comment:
Please make the fixed version 2.10.0. When testing locally, it is obvious that the vulnerability has not been patched, even if marked as such by both the advisory and bug tracker.